addons/l10n_pe/data/template/account.tax.group-pe.csv

```diff
@@ -11,4 +11,5 @@
 "tax_group_other","OTHERS","0","base.pe","OTROS"
 "tax_group_det","DET","100","base.pe","DET"
 "tax_group_icbper","ICBPER","0","base.pe",""
-"tax_group_ret","RET","100","base.pe","RET"
\ No newline at end of file
+"tax_group_ret","RET","100","base.pe","RET"
+"tax_group_free_invoice","Free Invoice","200","base.pe","Free Invoice"
```

Comment:
@desdelinux just small change pls: the last column is the Spanish translation, can you update it pls with "Factura Gratuita" instead pls.

Reply:
done